landing-pages/site/content/en/blog/airflow-survey.md

```diff
@@ -0,0 +1,333 @@
+|-------|---|------|
+|0-10   | 61|19.81%|
+|11-20  | 60|19.48%|
+|21-30  | 31|10.06%|
+|31-40  | 21|6.82% |
+|41-50  | 26|8.44% |
+|51-100 | 36|11.69%|
+|101-200| 28|9.09% |
+|201-500| 21|6.82% |
+|501+   | 24|11.54%|
+
+
+The given maximum number of tasks in a single DAG was 10 000 (!). Number of tasks
+depends on the purposes of a DAG so it’s rather hard to say if users have “simple”
+or “complicated” workflows.
+
+**When onboarding new members to Airflow, what is the biggest problem?**
+
+|                                                               |No.|  %   |
+|---------------------------------------------------------------|---|------|
+|No guide on best practises on developing DAGs                  |160|51.95%|
+|Small number of tutorials on different aspects of using Airflow| 57|18.51%|
+|Documentation is not clear enough                              | 42|13.64%|
+|Small number of blogs regarding Airflow                        |  6|1.95% |
+|Other                                                          | 43|13.96%|
+
+This is an important result. Using Airflow is all about writing and scheduling DAGs.
+No guide or any other complete resource on best practises for developing Dags is a big
+problem. Diving deep in the “other” answers we can find that:
+
+- Airflow’s “magic” (scheduler, executors, schedule times) is hard to understand
+DAG testing is not easy to do and to explain
+- Airflow UI needs some love.
+
+**How likely are you to recommend Apache Airflow?**
+
+|             |No.|  %   |
+|-------------|---|------|
+|Very Likely  |140|45.45%|
+|Likely       |124|40.26%|
+|Neutral      | 33|10.71%|
+|Unlikely     |  8|2.60% |
+|Very unlikely|  3|0.97% |
+
+This means that more than 85% of people who use Airflow like it. Seems Airflow does
+its job nicely. However, we have to remember that this survey is likely biased - it’s
+more likely that you respond to the survey if you like the tool you use. Should we
+focus then on those 8 people who did not like Airflow? It’s a good question.
+
+## Airflow usage
+
+**Which interface(s) of Airflow do you use as part of your current role?**
+
+|                                                     |No.|  %   |
+|-----------------------------------------------------|---|------|
+|Original Airflow Graphical User Interface            |297|96.43%|
+|CLI                                                  |126|40.91%|
+|Original Airflow Graphical User Interface, CLI       |117|37.99%|
+|API                                                  | 60|19.48%|
+|Original Airflow Graphical User Interface, CLI, API  | 32|10.39%|
+|Custom (own created) Airflow Graphical User Interface| 25|8.12% |
+
+It’s visible that usage of CLI goes in pair with using Airflow web UI. Our
+survey included some UX related questions to allow us to understand how users
+use Airflow webserver.
+
+**What do you use the Graphical User Interface for?**
+
+![](plot1.png)
+
+**What do you use CLI for?**
+
+![](plot2.png)
```

Comment:
Nit: the color palette in this plot is inverted compared to plot1.png
For example, blue goes from one opposite (very frequently) to the other (never).

Reply:
Good point, I will try to fix it